ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,61 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Tell us what happened.
+      placeholder: The app crashed when I...
+    validations:
+      required: true
+  - type: textarea
+    id: what-expected
+    attributes:
+      label: What should have happened?
+      description: Tell us what you expected to happen.
+      placeholder: When I ... the app should ...
+    validations:
+      required: true
+  - type: textarea
+    id: snap-info
+    attributes:
+      label: Output of `snap info $snap_name`
+      description: Please run `snap info $snap_name` (replace `$snap_name` with the name of this snap) in a terminal and paste the output here. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+    validations:
+      required: true
+  - type: textarea
+    id: snap-connections
+    attributes:
+      label: Output of `snap connections $snap_name`
+      description: Please run `snap connections $snap_name` (replace `$snap_name` with the name of this snap) in a terminal and paste the output here. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+    validations:
+      required: true
+  - type: textarea
+    id: snap-version
+    attributes:
+      label: Output of `snap version`
```

Comment:
Does `snap version` show the OS version too on non-ubuntu distros? Otherwise, everything looks good. +1 for merging this and testing it out.

Reply:
`snap version` does it's best to show the correct OS, yes.